client/views/room/contextualBar/RoomMembers/List/RoomMembersWithData.js

```diff
@@ -20,6 +20,7 @@ const RoomMembersWithData = ({ rid }) => {
 	const [state, setState] = useState({});
 	const onClickClose = useTabBarClose();
 	const room = useUserRoom(rid);
+	const isTeam = room.teamId;
```

Comment:
I think you're doing the wrong check here. If the room is a team, it should have `teamMain`. If it only has a teamId, it could be just a room inside a team.

Reply:
great observation. I made the changes on the toolbox as well, thanks!